cheatsheets/Docker_Security_Cheat_Sheet.md

```diff
@@ -235,13 +235,7 @@ To detect misconfigurations in Docker:
 
 ## RULE \#10 - Set the logging level to at least INFO
 
-Set Docker daemon log level to 'info'. Rationale: Setting up an appropriate log level, configures the Docker daemon to log events that you would want to review later. A ase log level of 'info' and above would capture all logs except debug logs. Until and unless required, you should not run docker daemon at 'debug' log level.
-
-To configure the log level per container:
-
-```
-$ docker --log-level info run -it alpine
-```
+By default, the Docker daemon is configured to have a base logging level of 'info', if this is not the case: set the Docker daemon log level to 'info'. Rationale: Setting up an appropriate log level, configures the Docker daemon to log events that you would want to review later. A ase log level of 'info' and above would capture all logs except debug logs. Until and unless required, you should not run docker daemon at 'debug' log level.
```

Comment:
I'd improve the sentence by saying `'info, and if this is not ...`, Add the `and`. And please, fix the typo: `A base log`, where it says `A ase log`.

Reply:
Fixed in commit b182e99